src/main/java/seedu/address/storage/JsonTranscriptStorage.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.storage;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import seedu.address.commons.exceptions.DataConversionException;
+import seedu.address.commons.util.FileUtil;
+import seedu.address.commons.util.JsonUtil;
+import seedu.address.model.ReadOnlyTranscript;
+import seedu.address.model.Transcript;
+
+/**
+ * A class to access Transcript stored in the hard disk as a json file
+ */
+public class JsonTranscriptStorage implements TranscriptStorage {
+
+    private Path filePath;
```

Comment:
Consider making it final if its not mutable. Helps to prevent making mistakes

Reply:
Done!